docs/writing_wiki.md

```diff
@@ -5,11 +5,86 @@ title: "Editing the Wiki"
 # Editing the wiki
 
 ## MkDocs
+
 The wiki is built with [MkDocs](https://www.mkdocs.org/), using the [Material theme](https://squidfunk.github.io/mkdocs-material/).
 
 MkDocs is configured in the [mkdocs.yml](https://github.com/aau-giraf/wiki/blob/master/mkdocs.yml) 
 
 The following plugins are used:
 
+* search
 * [awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin)
+* [git-revision-date-localized](https://github.com/timvink/mkdocs-git-revision-date-localized-plugin)
+
+If more plugins are downloaded with pip, remember to add them to ``requirements.txt``
+
+### Running locally
+
+The wiki can be run locally by doing the following:
+
+* Install python 3
+    * check that pip is using python3 with ``pip -V``
+    * if not, check ``pip3 -V``
+* ``pip install -r requirements.txt`` (if pip is using python3)
+    * else ``pip3 install -r requirements.txt``
+* ``mkdocs serve``
```

Comment:
Can there be added a little more words. Is the "pip3 install -r requirements.txt" command supposed to be done by the command window or in Python? Also what dores mkdocs serve mean?

Reply:
> 
> 
> Can there be added a little more words. Is the "pip3 install -r requirements.txt" command supposed to be done by the command window or in Python? Also what dores mkdocs serve mean?


See newest commit (3b8d9e1) is this better? :)